c/CMakeLists.txt

```diff
@@ -1,7 +1,21 @@
 
 set(CMAKE_C_STANDARD 99)
 
-add_library (primac cintrf.f90 cobyla_c.f90 lincoa_c.f90 bobyqa_c.f90 newuoa_c.f90 uobyqa_c.f90 prima.c)
+include_directories(include)
+
+if (WIN32 AND BUILD_SHARED_LIBS)
+  add_compile_definitions(PRIMAC_EXPORTS)
```

Comment:
Need the macro to enable `__declspec(dllexport)`.

Reply:
Because now `add_library(primac_c OBJECT prima.c)`, `target_EXPORT` is not defined, this emits inconsistent dll linkage warning C4273.